docs/QUALITY_CHECKLIST.md

```diff
@@ -0,0 +1,57 @@
+The purpose of this checklist is to assess the state of this repository, and bring it to an acceptable level for long-term support by completing the tasks listed below.
+
+> Please check every part of this checklist for us to process your request. You can do this by marking them with [x] instead of [ ].
+
+### Documentation
+
+- [ ] Twilio [LICENSE](https://github.com/twilio-labs/code-exchange/blob/master/LICENSE) exists in repository
+
+- [ ] [Code of Conduct](https://github.com/twilio-labs/.github/blob/master/CODE_OF_CONDUCT.md) exists in repository
+
+- [ ] [CONTRIBUTING](https://github.com/twilio-labs/.github/blob/master/CONTRIBUTING.md) exists in repository 
+
+- [ ] No broken links in README.md
+
+- [ ] Build status exists at top of README.md
+
+- [ ] Set up steps are complete and accurate
+
+- [ ] CodeExchange repository is linked in README.md
+
+### GitHub
+
+- [ ] Dependabot set up
+
+- [ ] Dependabot automerge set up
+
+- [ ] Uses GitHub Actions (required for Twilio sample) or other CI
+
+### Code
+
+- [ ] Packages/dependencies are up to date
+
+- [ ] Code compiles against latest packages/dependencies
+
+- [ ] Matches [CodeExchange Guidelines](https://github.com/twilio-labs/code-exchange/tree/master/guidelines) to the best of ability
+
+### Tests
+
+- [ ] Functionality tests have been created
+
+- [ ] All tests pass
+
+### App
+
+- [ ] Confirmed sample behaves as expected (after dependencies have been updated)
+
+- [ ] This file, QUALITY_CHECKLIST.md, exists in repository
```

Comment:
Maybe this should be something like `This file has been copied into a wiki entry for the respective repository or added as a GitHub issue`

Reply:
👍